src/components/Modal/Modal.tsx

```diff
@@ -7,7 +7,7 @@ import { faClose } from "@fortawesome/free-solid-svg-icons";
 import Icon from "@/components/Icons/Icon";
 
 export const StyledDialog = styled(Dialog)`
-    & .MuiPaper-root {
+    & .MuiPaper-root:first-of-type {
```

Comment:
This seemed to be intended to modify the style of the main modal box, but actually selected all `MuiPaper-root` classes inside the modal. This was messing with the styling of my `Alert` so I changed this to just select the first `MuiPaper-root` class, which should just affect the modal itself now.